
Property changes on: solr
___________________________________________________________________
Modified: svn:ignore
   - build
dist
.project
.classpath
.settings
bin
target
luke
lucene-libs
build.properties

   + build
dist
.project
.classpath
.settings
bin
target
luke
lucene-libs
build.properties
*.iml



Property changes on: solr/contrib/clustering
___________________________________________________________________
Modified: svn:ignore
   - build

   + build
*.iml



Property changes on: solr/contrib/dataimporthandler
___________________________________________________________________
Modified: svn:ignore
   - target

   + target
*.iml



Property changes on: solr/contrib/dataimporthandler/src/extras
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: solr/contrib/analysis-extras
___________________________________________________________________
Modified: svn:ignore
   - build
lucene-libs

   + build
lucene-libs
*.iml



Property changes on: lucene/contrib/icu
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/wordnet
___________________________________________________________________
Modified: svn:ignore
   - 

   + *.iml



Property changes on: lucene/contrib/ant
___________________________________________________________________
Modified: svn:ignore
   - 

   + *.iml



Property changes on: lucene/contrib/queryparser
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/demo
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/db/bdb-je
___________________________________________________________________
Modified: svn:ignore
   - lib
index

   + lib
index
*.iml



Property changes on: lucene/contrib/db/bdb
___________________________________________________________________
Modified: svn:ignore
   - lib
index

   + lib
index
*.iml



Property changes on: lucene/contrib/swing
___________________________________________________________________
Modified: svn:ignore
   - 

   + *.iml



Property changes on: lucene/contrib/instantiated
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/remote
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/misc
___________________________________________________________________
Modified: svn:ignore
   - 

   + *.iml



Property changes on: lucene/contrib/xml-query-parser
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/spatial
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/highlighter
___________________________________________________________________
Modified: svn:ignore
   - 

   + *.iml



Property changes on: lucene/contrib/spellchecker
___________________________________________________________________
Modified: svn:ignore
   - 

   + *.iml



Property changes on: lucene/contrib/memory
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/lucli
___________________________________________________________________
Modified: svn:ignore
   - 

   + *.iml



Property changes on: lucene/contrib/analyzers/smartcn
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/analyzers/common
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/analyzers/stempel
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/contrib/queries
___________________________________________________________________
Added: svn:ignore
   + *.iml


Index: dev-tools/idea/solr/solr.iml
===================================================================
--- dev-tools/idea/solr/solr.iml	(revision 0)
+++ dev-tools/idea/solr/solr.iml	(revision 0)
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/build/solr" />
+    <output-test url="file://$MODULE_DIR$/build/tests" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/solrj" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/common" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/webapp/src" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/webapp/web" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/scripts" isTestSource="false" />
+      <excludeFolder url="file://$MODULE_DIR$/build" />
+    </content>
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="highlighter" />
+    <orderEntry type="module" module-name="memory" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="spatial" />
+    <orderEntry type="module" module-name="spellchecker" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Solr library" level="project" />
+    <orderEntry type="library" name="Solr example library" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/solr/contrib/clustering/clustering.iml
===================================================================
--- dev-tools/idea/solr/contrib/clustering/clustering.iml	(revision 0)
+++ dev-tools/idea/solr/contrib/clustering/clustering.iml	(revision 0)
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/build/classes" />
+    <output-test url="file://$MODULE_DIR$/build/test-classes" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/resources" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="solr" />
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="highlighter" />
+    <orderEntry type="module" module-name="memory" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="spatial" />
+    <orderEntry type="module" module-name="spellchecker" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" name="Solr library" level="project" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/solr/contrib/extraction/extraction.iml
===================================================================
--- dev-tools/idea/solr/contrib/extraction/extraction.iml	(revision 0)
+++ dev-tools/idea/solr/contrib/extraction/extraction.iml	(revision 0)
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/build/classes" />
+    <output-test url="file://$MODULE_DIR$/build/test-classes" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/resources" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
+      <excludeFolder url="file://$MODULE_DIR$/build" />
+    </content>
+    <orderEntry type="module" module-name="solr" />
+    <orderEntry type="module" module-name="lucene" scope="TEST" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Solr library" level="project" />
+    <orderEntry type="library" name="Solr extraction library" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/solr/contrib/dataimporthandler/dataimporthandler.iml
===================================================================
--- dev-tools/idea/solr/contrib/dataimporthandler/dataimporthandler.iml	(revision 0)
+++ dev-tools/idea/solr/contrib/dataimporthandler/dataimporthandler.iml	(revision 0)
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/target/classes" />
+    <output-test url="file://$MODULE_DIR$/target/test-classes" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/main/webapp" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/resources" isTestSource="true" />
+      <excludeFolder url="file://$MODULE_DIR$/target" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="solr" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Solr library" level="project" />
+    <orderEntry type="library" name="Solr DIH library" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/solr/contrib/dataimporthandler/src/extras/extras.iml
===================================================================
--- dev-tools/idea/solr/contrib/dataimporthandler/src/extras/extras.iml	(revision 0)
+++ dev-tools/idea/solr/contrib/dataimporthandler/src/extras/extras.iml	(revision 0)
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../target/extras/classes" />
+    <output-test url="file://$MODULE_DIR$/../../target/extras/test-classes" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/main/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/test/java" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/test/resources" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="solr" />
+    <orderEntry type="module" module-name="dataimporthandler" />
+    <orderEntry type="module" module-name="lucene" scope="TEST" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Solr library" level="project" />
+    <orderEntry type="library" name="Solr DIH library" level="project" />
+    <orderEntry type="library" name="Solr extraction library" level="project" />
+  </component>
+</module>
+
Index: dev-tools/idea/solr/contrib/analysis-extras/analysis-extras.iml
===================================================================
--- dev-tools/idea/solr/contrib/analysis-extras/analysis-extras.iml	(revision 0)
+++ dev-tools/idea/solr/contrib/analysis-extras/analysis-extras.iml	(revision 0)
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/build/classes" />
+    <output-test url="file://$MODULE_DIR$/build/test-classes" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="icu" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="smartcn" />
+    <orderEntry type="module" module-name="solr" />
+    <orderEntry type="module" module-name="stempel" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" name="Solr library" level="project" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/.idea/ant.xml
===================================================================
--- dev-tools/idea/.idea/ant.xml	(revision 0)
+++ dev-tools/idea/.idea/ant.xml	(revision 0)
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="AntConfiguration">
+    <buildFile url="file://$PROJECT_DIR$/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/analyzers/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/analyzers/common/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/analyzers/smartcn/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/analyzers/stempel/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/ant/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/benchmark/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/db/bdb/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/db/bdb-je/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/db/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/highlighter/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/icu/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/instantiated/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/lucli/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/memory/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/misc/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/queries/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/queryparser/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/remote/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/spatial/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/spellchecker/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/swing/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/wordnet/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/lucene/contrib/xml-query-parser/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/solr/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/solr/contrib/analysis-extras/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/solr/contrib/clustering/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/solr/contrib/dataimporthandler/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/solr/contrib/extraction/build.xml" />
+  </component>
+</project>
+

Property changes on: dev-tools/idea/.idea/ant.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/modules.xml
===================================================================
--- dev-tools/idea/.idea/modules.xml	(revision 0)
+++ dev-tools/idea/.idea/modules.xml	(revision 0)
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module filepath="$PROJECT_DIR$/solr/contrib/analysis-extras/analysis-extras.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/ant/ant.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/db/bdb/bdb.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/db/bdb-je/bdb-je.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/benchmark/benchmark.iml" />
+      <module filepath="$PROJECT_DIR$/solr/contrib/clustering/clustering.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/analyzers/common/common.iml" />
+      <module filepath="$PROJECT_DIR$/solr/contrib/dataimporthandler/dataimporthandler.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/demo/demo.iml" />
+      <module filepath="$PROJECT_DIR$/solr/contrib/extraction/extraction.iml" />
+      <module filepath="$PROJECT_DIR$/solr/contrib/dataimporthandler/src/extras/extras.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/highlighter/highlighter.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/icu/icu.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/instantiated/instantiated.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/lucene.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/lucli/lucli.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/memory/memory.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/misc/misc.iml" />
+      <module filepath="$PROJECT_DIR$/parent.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/queries/queries.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/queryparser/queryparser.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/remote/remote.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/analyzers/smartcn/smartcn.iml" />
+      <module filepath="$PROJECT_DIR$/solr/solr.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/spatial/spatial.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/spellchecker/spellchecker.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/analyzers/stempel/stempel.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/swing/swing.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/wordnet/wordnet.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/contrib/xml-query-parser/xml-query-parser.iml" />
+    </modules>
+  </component>
+</project>

Property changes on: dev-tools/idea/.idea/modules.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/compiler.xml
===================================================================
--- dev-tools/idea/.idea/compiler.xml	(revision 0)
+++ dev-tools/idea/.idea/compiler.xml	(revision 0)
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
+    <resourceExtensions>
+      <entry name=".+\.(properties|xml|html|dtd|tld)" />
+      <entry name=".+\.(gif|png|jpeg|jpg)" />
+    </resourceExtensions>
+    <wildcardResourcePatterns>
+      <entry name="?*.properties" />
+      <entry name="?*.xml" />
+      <entry name="?*.gif" />
+      <entry name="?*.png" />
+      <entry name="?*.jpeg" />
+      <entry name="?*.jpg" />
+      <entry name="?*.html" />
+      <entry name="?*.dtd" />
+      <entry name="?*.tld" />
+      <entry name="?*.ftl" />
+      <entry name="?*.txt" />
+      <entry name="?*.bz2" />
+      <entry name="?*.gz" />
+      <entry name="?*.htm" />
+      <entry name="?*.zip" />
+      <entry name="?*.nrm" />
+      <entry name="?*.brk" />
+      <entry name="?*.mem" />
+      <entry name="?*.tbl" />
+      <entry name="?*.xsl" />
+    </wildcardResourcePatterns>
+  </component>
+</project>
+

Property changes on: dev-tools/idea/.idea/compiler.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/vcs.xml
===================================================================
--- dev-tools/idea/.idea/vcs.xml	(revision 0)
+++ dev-tools/idea/.idea/vcs.xml	(revision 0)
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="svn" />
+  </component>
+</project>
+

Property changes on: dev-tools/idea/.idea/vcs.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/libraries/Servlet_API_2_4.xml
===================================================================
--- dev-tools/idea/.idea/libraries/Servlet_API_2_4.xml	(revision 0)
+++ dev-tools/idea/.idea/libraries/Servlet_API_2_4.xml	(revision 0)
@@ -0,0 +1,9 @@
+<component name="libraryTable">
+  <library name="Servlet API 2.4">
+    <CLASSES>
+      <root url="jar://$PROJECT_DIR$/lucene/contrib/demo/lib/servlet-api-2.4.jar!/" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+  </library>
+</component>

Property changes on: dev-tools/idea/.idea/libraries/Servlet_API_2_4.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/libraries/Ant.xml
===================================================================
--- dev-tools/idea/.idea/libraries/Ant.xml	(revision 0)
+++ dev-tools/idea/.idea/libraries/Ant.xml	(revision 0)
@@ -0,0 +1,9 @@
+<component name="libraryTable">
+  <library name="Ant">
+    <CLASSES>
+      <root url="jar://$PROJECT_DIR$/lucene/lib/ant-1.7.1.jar!/" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+  </library>
+</component>

Property changes on: dev-tools/idea/.idea/libraries/Ant.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/libraries/Solr_extraction_library.xml
===================================================================
--- dev-tools/idea/.idea/libraries/Solr_extraction_library.xml	(revision 0)
+++ dev-tools/idea/.idea/libraries/Solr_extraction_library.xml	(revision 0)
@@ -0,0 +1,10 @@
+<component name="libraryTable">
+  <library name="Solr extraction library">
+    <CLASSES>
+      <root url="file://$PROJECT_DIR$/solr/contrib/extraction/lib" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+    <jarDirectory url="file://$PROJECT_DIR$/solr/contrib/extraction/lib" recursive="false" />
+  </library>
+</component>

Property changes on: dev-tools/idea/.idea/libraries/Solr_extraction_library.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/libraries/Solr_library.xml
===================================================================
--- dev-tools/idea/.idea/libraries/Solr_library.xml	(revision 0)
+++ dev-tools/idea/.idea/libraries/Solr_library.xml	(revision 0)
@@ -0,0 +1,10 @@
+<component name="libraryTable">
+  <library name="Solr library">
+    <CLASSES>
+      <root url="file://$PROJECT_DIR$/solr/lib" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+    <jarDirectory url="file://$PROJECT_DIR$/solr/lib" recursive="false" />
+  </library>
+</component>
\ No newline at end of file

Property changes on: dev-tools/idea/.idea/libraries/Solr_library.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/libraries/Solr_DIH_library.xml
===================================================================
--- dev-tools/idea/.idea/libraries/Solr_DIH_library.xml	(revision 0)
+++ dev-tools/idea/.idea/libraries/Solr_DIH_library.xml	(revision 0)
@@ -0,0 +1,10 @@
+<component name="libraryTable">
+  <library name="Solr DIH library">
+    <CLASSES>
+      <root url="file://$PROJECT_DIR$/solr/contrib/dataimporthandler/lib" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+    <jarDirectory url="file://$PROJECT_DIR$/solr/contrib/dataimporthandler/lib" recursive="false" />
+  </library>
+</component>
\ No newline at end of file

Property changes on: dev-tools/idea/.idea/libraries/Solr_DIH_library.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/libraries/JUnit.xml
===================================================================
--- dev-tools/idea/.idea/libraries/JUnit.xml	(revision 0)
+++ dev-tools/idea/.idea/libraries/JUnit.xml	(revision 0)
@@ -0,0 +1,9 @@
+<component name="libraryTable">
+  <library name="JUnit">
+    <CLASSES>
+      <root url="jar://$PROJECT_DIR$/lucene/lib/junit-4.7.jar!/" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+  </library>
+</component>

Property changes on: dev-tools/idea/.idea/libraries/JUnit.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/libraries/Solr_example_library.xml
===================================================================
--- dev-tools/idea/.idea/libraries/Solr_example_library.xml	(revision 0)
+++ dev-tools/idea/.idea/libraries/Solr_example_library.xml	(revision 0)
@@ -0,0 +1,10 @@
+<component name="libraryTable">
+  <library name="Solr example library">
+    <CLASSES>
+      <root url="file://$PROJECT_DIR$/solr/example/lib" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+    <jarDirectory url="file://$PROJECT_DIR$/solr/example/lib" recursive="true" />
+  </library>
+</component>
\ No newline at end of file

Property changes on: dev-tools/idea/.idea/libraries/Solr_example_library.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/.idea/workspace.xml
===================================================================
--- dev-tools/idea/.idea/workspace.xml	(revision 0)
+++ dev-tools/idea/.idea/workspace.xml	(revision 0)
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RunManager">
+    <configuration default="false" name="analysis-extras contrib" type="JUnit" factoryName="JUnit">
+      <module name="analysis-extras" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=3.1 -DtempDir=../../../../../build/test-results" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/contrib/analysis-extras/src/test/test-files" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="ant contrib" type="JUnit" factoryName="JUnit">
+      <module name="ant" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/ant" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/ant/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="bdb contrib" type="JUnit" factoryName="JUnit">
+      <module name="bdb" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/db/bdb" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/db/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="bdb-je contrib" type="JUnit" factoryName="JUnit">
+      <module name="bdb-je" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/db/bdb-je" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/db/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="benchmark contrib" type="JUnit" factoryName="JUnit">
+      <module name="benchmark" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/benchmark" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/benchmark/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="clustering contrib" type="JUnit" factoryName="JUnit">
+      <module name="clustering" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=3.1 -DtempDir=../../../../../build/test-results" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/contrib/clustering/src/test/resources" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="common analyzer contrib" type="JUnit" factoryName="JUnit">
+      <module name="common" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/analyzers/common" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../../build/contrib/analyzers/common/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="dataimporthandler contrib" type="JUnit" factoryName="JUnit">
+      <module name="dataimporthandler" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/contrib/dataimporthandler/src/test/resources" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../../../../build/test-results/temp" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="demo contrib" type="JUnit" factoryName="JUnit">
+      <module name="demo" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/demo" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/demo/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="extraction contrib" type="JUnit" factoryName="JUnit">
+      <module name="extraction" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/contrib/extraction/src/test/resources" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../../../../build/test-results/temp" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="extras from dataimporthandler contrib" type="JUnit" factoryName="JUnit">
+      <module name="extras" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/contrib/dataimporthandler/src/extras/test/resources" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../../../../../build/test-results/temp" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="highlighter contrib" type="JUnit" factoryName="JUnit">
+      <module name="highlighter" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/highlighter" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/highlighter/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="icu contrib" type="JUnit" factoryName="JUnit">
+      <module name="icu" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/icu" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/icu/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="instantiated contrib" type="JUnit" factoryName="JUnit">
+      <module name="instantiated" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/instantiated" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/instantiated/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="lucene" type="JUnit" factoryName="JUnit">
+      <module name="lucene" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=build/test -Dlucene.version=3.1-SNAPSHOT -Dtests.linedocsfile=europarl.lines.txt.gz" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="lucli contrib" type="JUnit" factoryName="JUnit">
+      <module name="lucli" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/lucli" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/lucli/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="memory contrib" type="JUnit" factoryName="JUnit">
+      <module name="memory" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/memory" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/memory/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="misc contrib" type="JUnit" factoryName="JUnit">
+      <module name="misc" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/misc" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/misc/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="queries contrib" type="JUnit" factoryName="JUnit">
+      <module name="queries" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/queries" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/queries/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="queryparser contrib" type="JUnit" factoryName="JUnit">
+      <module name="queryparser" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/queryparser" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/queryparser/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="remote contrib" type="JUnit" factoryName="JUnit">
+      <module name="remote" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/remote" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/remote/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="smartcn analyzer contrib" type="JUnit" factoryName="JUnit">
+      <module name="smartcn" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/analyzers/smartcn" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/analyzers/smartcn/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="solr" type="JUnit" factoryName="JUnit">
+      <module name="solr" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=3.1 -DtempDir=../../../build/test-results/temp" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/src/test/test-files" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="spatial contrib" type="JUnit" factoryName="JUnit">
+      <module name="spatial" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/spatial" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/spatial/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="spellchecker contrib" type="JUnit" factoryName="JUnit">
+      <module name="spellchecker" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/spellchecker" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/spellchecker/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="stempel analyzer contrib" type="JUnit" factoryName="JUnit">
+      <module name="stempel" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/analyzers/stempel" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/analyzers/stempel/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="swing contrib" type="JUnit" factoryName="JUnit">
+      <module name="swing" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/swing" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/swing/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="wordnet contrib" type="JUnit" factoryName="JUnit">
+      <module name="wordnet" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/wordnet" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/wordnet/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <configuration default="false" name="xml-query-parser contrib" type="JUnit" factoryName="JUnit">
+      <module name="xml-query-parser" />
+      <option name="TEST_OBJECT" value="package" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/contrib/xml-query-parser" />
+      <option name="VM_PARAMETERS" value="-ea -DtempDir=../../build/contrib/xml-query-parser/test" />
+      <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
+    </configuration>
+    <list size="28">
+      <item index="0" class="java.lang.String" itemvalue="JUnit.analysis-extras contrib" />
+      <item index="1" class="java.lang.String" itemvalue="JUnit.ant contrib" />
+      <item index="2" class="java.lang.String" itemvalue="JUnit.bdb contrib" />
+      <item index="3" class="java.lang.String" itemvalue="JUnit.bdb-je contrib" />
+      <item index="4" class="java.lang.String" itemvalue="JUnit.benchmark contrib" />
+      <item index="5" class="java.lang.String" itemvalue="JUnit.clustering contrib" />
+      <item index="6" class="java.lang.String" itemvalue="JUnit.common analyzer contrib" />
+      <item index="7" class="java.lang.String" itemvalue="JUnit.dataimporthandler contrib" />
+      <item index="8" class="java.lang.String" itemvalue="JUnit.extraction contrib" />
+      <item index="9" class="java.lang.String" itemvalue="JUnit.extras from dataimporthandler contrib" />
+      <item index="10" class="java.lang.String" itemvalue="JUnit.highlighter contrib" />
+      <item index="11" class="java.lang.String" itemvalue="JUnit.icu contrib" />
+      <item index="12" class="java.lang.String" itemvalue="JUnit.instantiated contrib" />
+      <item index="13" class="java.lang.String" itemvalue="JUnit.lucene" />
+      <item index="14" class="java.lang.String" itemvalue="JUnit.lucli contrib" />
+      <item index="15" class="java.lang.String" itemvalue="JUnit.memory contrib" />
+      <item index="16" class="java.lang.String" itemvalue="JUnit.misc contrib" />
+      <item index="17" class="java.lang.String" itemvalue="JUnit.queries contrib" />
+      <item index="18" class="java.lang.String" itemvalue="JUnit.queryparser contrib" />
+      <item index="19" class="java.lang.String" itemvalue="JUnit.remote contrib" />
+      <item index="20" class="java.lang.String" itemvalue="JUnit.smartcn analyzer contrib" />
+      <item index="21" class="java.lang.String" itemvalue="JUnit.solr" />
+      <item index="22" class="java.lang.String" itemvalue="JUnit.spatial contrib" />
+      <item index="23" class="java.lang.String" itemvalue="JUnit.spellchecker contrib" />
+      <item index="24" class="java.lang.String" itemvalue="JUnit.stempel analyzer contrib" />
+      <item index="25" class="java.lang.String" itemvalue="JUnit.swing contrib" />
+      <item index="26" class="java.lang.String" itemvalue="JUnit.wordnet contrib" />
+      <item index="27" class="java.lang.String" itemvalue="JUnit.xml-query-parser contrib" />
+    </list>
+  </component>
+</project>

Property changes on: dev-tools/idea/.idea/workspace.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/parent.iml
===================================================================
--- dev-tools/idea/parent.iml	(revision 0)
+++ dev-tools/idea/parent.iml	(revision 0)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
+
Index: dev-tools/idea/lucene/contrib/icu/icu.iml
===================================================================
--- dev-tools/idea/lucene/contrib/icu/icu.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/icu/icu.iml	(revision 0)
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../build/icu/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../build/icu/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/tools/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/resources" isTestSource="false" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/wordnet/wordnet.iml
===================================================================
--- dev-tools/idea/lucene/contrib/wordnet/wordnet.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/wordnet/wordnet.iml	(revision 0)
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/wordnet/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/wordnet/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/ant/ant.iml
===================================================================
--- dev-tools/idea/lucene/contrib/ant/ant.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/ant/ant.iml	(revision 0)
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/ant/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/ant/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" name="Ant" level="project" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/queryparser/queryparser.iml
===================================================================
--- dev-tools/idea/lucene/contrib/queryparser/queryparser.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/queryparser/queryparser.iml	(revision 0)
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/queryparser/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/queryparser/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/demo/demo.iml
===================================================================
--- dev-tools/idea/lucene/contrib/demo/demo.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/demo/demo.iml	(revision 0)
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/demo/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/demo/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/jsp" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/db/bdb-je/bdb-je.iml
===================================================================
--- dev-tools/idea/lucene/contrib/db/bdb-je/bdb-je.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/db/bdb-je/bdb-je.iml	(revision 0)
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../../build/contrib/db/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../build/contrib/db/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/db/bdb/bdb.iml
===================================================================
--- dev-tools/idea/lucene/contrib/db/bdb/bdb.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/db/bdb/bdb.iml	(revision 0)
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../../build/contrib/db/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../build/contrib/db/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/swing/swing.iml
===================================================================
--- dev-tools/idea/lucene/contrib/swing/swing.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/swing/swing.iml	(revision 0)
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/swing/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/swing/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/instantiated/instantiated.iml
===================================================================
--- dev-tools/idea/lucene/contrib/instantiated/instantiated.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/instantiated/instantiated.iml	(revision 0)
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/instantiated/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/instantiated/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/benchmark/benchmark.iml
===================================================================
--- dev-tools/idea/lucene/contrib/benchmark/benchmark.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/benchmark/benchmark.iml	(revision 0)
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/benchmark/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/benchmark/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/scripts" isTestSource="false" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/work" />
+    </content>
+    <orderEntry type="module" module-name="highlighter" />
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="icu" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="module" module-name="memory" />
+    <orderEntry type="module" module-name="demo" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
+
Index: dev-tools/idea/lucene/contrib/misc/misc.iml
===================================================================
--- dev-tools/idea/lucene/contrib/misc/misc.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/misc/misc.iml	(revision 0)
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/misc/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/misc/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+    </content>
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/remote/remote.iml
===================================================================
--- dev-tools/idea/lucene/contrib/remote/remote.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/remote/remote.iml	(revision 0)
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/remote/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/remote/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+    </content>
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/xml-query-parser/xml-query-parser.iml
===================================================================
--- dev-tools/idea/lucene/contrib/xml-query-parser/xml-query-parser.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/xml-query-parser/xml-query-parser.iml	(revision 0)
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/xml-query-parser/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/xml-query-parser/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/demo/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/demo/WebContent" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+    <orderEntry type="library" name="Servlet API 2.4" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/spatial/spatial.iml
===================================================================
--- dev-tools/idea/lucene/contrib/spatial/spatial.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/spatial/spatial.iml	(revision 0)
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/spatial/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/spatial/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
+
Index: dev-tools/idea/lucene/contrib/highlighter/highlighter.iml
===================================================================
--- dev-tools/idea/lucene/contrib/highlighter/highlighter.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/highlighter/highlighter.iml	(revision 0)
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/highlighter/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/highlighter/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="memory" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/spellchecker/spellchecker.iml
===================================================================
--- dev-tools/idea/lucene/contrib/spellchecker/spellchecker.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/spellchecker/spellchecker.iml	(revision 0)
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/spellchecker/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/spellchecker/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/memory/memory.iml
===================================================================
--- dev-tools/idea/lucene/contrib/memory/memory.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/memory/memory.iml	(revision 0)
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/memory/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/memory/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/lucli/lucli.iml
===================================================================
--- dev-tools/idea/lucene/contrib/lucli/lucli.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/lucli/lucli.iml	(revision 0)
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/lucli/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/lucli/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/analyzers/smartcn/smartcn.iml
===================================================================
--- dev-tools/idea/lucene/contrib/analyzers/smartcn/smartcn.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/analyzers/smartcn/smartcn.iml	(revision 0)
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../../build/analyzers/smartcn/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../build/analyzers/smartcn/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/resources" isTestSource="false" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/analyzers/common/common.iml
===================================================================
--- dev-tools/idea/lucene/contrib/analyzers/common/common.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/analyzers/common/common.iml	(revision 0)
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../../build/contrib/analyzers/common/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../build/contrib/analyzers/common/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/resources" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="remote" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="queries" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/analyzers/stempel/stempel.iml
===================================================================
--- dev-tools/idea/lucene/contrib/analyzers/stempel/stempel.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/analyzers/stempel/stempel.iml	(revision 0)
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../../build/analyzers/stempel/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../build/analyzers/stempel/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/resources" isTestSource="false" />
+    </content>
+    <orderEntry type="module" module-name="common" />
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/contrib/queries/queries.iml
===================================================================
--- dev-tools/idea/lucene/contrib/queries/queries.iml	(revision 0)
+++ dev-tools/idea/lucene/contrib/queries/queries.iml	(revision 0)
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../build/contrib/queries/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../build/contrib/queries/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+    </content>
+    <orderEntry type="module" module-name="lucene" />
+    <orderEntry type="module" module-name="misc" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="jar://$MODULE_DIR$/lib/jakarta-regexp-1.4.jar!/" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+      </library>
+    </orderEntry>
+    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/lucene.iml
===================================================================
--- dev-tools/idea/lucene/lucene.iml	(revision 0)
+++ dev-tools/idea/lucene/lucene.iml	(revision 0)
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/build/classes/java" />
+    <output-test url="file://$MODULE_DIR$/build/classes/test" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/demo" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/jsp" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
+      <excludeFolder url="file://$MODULE_DIR$/build" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library" scope="TEST">
+      <library>
+        <CLASSES>
+          <root url="file://$MODULE_DIR$/lib" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+      </library>
+    </orderEntry>
+  </component>
+</module>
+
Index: dev-tools/idea/Intellij-Lucene-Codestyle.xml
===================================================================
--- dev-tools/idea/Intellij-Lucene-Codestyle.xml	(revision 0)
+++ dev-tools/idea/Intellij-Lucene-Codestyle.xml	(revision 0)
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<code_scheme name="Lucene" parent="Default">
+  <option name="JAVA_INDENT_OPTIONS">
+    <value>
+      <option name="INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="8" />
+      <option name="TAB_SIZE" value="2" />
+      <option name="USE_TAB_CHARACTER" value="false" />
+      <option name="SMART_TABS" value="false" />
+      <option name="LABEL_INDENT_SIZE" value="0" />
+      <option name="LABEL_INDENT_ABSOLUTE" value="false" />
+    </value>
+  </option>
+  <option name="JSP_INDENT_OPTIONS">
+    <value>
+      <option name="INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="8" />
+      <option name="TAB_SIZE" value="2" />
+      <option name="USE_TAB_CHARACTER" value="false" />
+      <option name="SMART_TABS" value="false" />
+      <option name="LABEL_INDENT_SIZE" value="0" />
+      <option name="LABEL_INDENT_ABSOLUTE" value="false" />
+    </value>
+  </option>
+  <option name="XML_INDENT_OPTIONS">
+    <value>
+      <option name="INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="8" />
+      <option name="TAB_SIZE" value="2" />
+      <option name="USE_TAB_CHARACTER" value="false" />
+      <option name="SMART_TABS" value="false" />
+      <option name="LABEL_INDENT_SIZE" value="0" />
+      <option name="LABEL_INDENT_ABSOLUTE" value="false" />
+    </value>
+  </option>
+  <option name="OTHER_INDENT_OPTIONS">
+    <value>
+      <option name="INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="8" />
+      <option name="TAB_SIZE" value="2" />
+      <option name="USE_TAB_CHARACTER" value="false" />
+      <option name="SMART_TABS" value="false" />
+      <option name="LABEL_INDENT_SIZE" value="0" />
+      <option name="LABEL_INDENT_ABSOLUTE" value="false" />
+    </value>
+  </option>
+</code_scheme>
+

Property changes on: dev-tools/idea/Intellij-Lucene-Codestyle.xml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: build.xml
===================================================================
--- build.xml	(revision 1053983)
+++ build.xml	(working copy)
@@ -4,7 +4,7 @@
     Licensed to the Apache Software Foundation (ASF) under one or more
     contributor license agreements.  See the NOTICE file distributed with
     this work for additional information regarding copyright ownership.
-    The ASF licenses this file to You under the Apache License, Version 2.0
+    The ASF licenses this file to You under the Apache Licens2e, Version 2.0
     the "License"); you may not use this file except in compliance with
     the License.  You may obtain a copy of the License at
  
@@ -51,6 +51,81 @@
     </sequential>
   </target>
 
+  <condition property="isMac"><os family="mac"/></condition>
+  <condition property="isUnix"><os family="unix"/></condition>
+  <condition property="isWindows"><os family="windows"/></condition>
+  <property environment="env"/>
+  <target name="idea" depends="copy-idea-files,finish-idea-setup-windows,
+                               finish-idea-setup-mac,
+                               finish-idea-setup-unix"/>
+  <target name="copy-idea-files" 
+          description="Setup IntelliJ IDEA configuration files">
+    <copy todir=".">
+      <fileset dir="dev-tools/idea">
+        <exclude name="Intellij-Lucene-Codestyle.xml"/>
+      </fileset>
+    </copy>
+    <echo>Running Lucene contrib db/bdb-je task 'get-je-jar' ...</echo>
+    <subant target="get-je-jar">
+      <fileset dir="lucene/contrib/db/bdb-je" includes="build.xml" />
+    </subant>
+    <echo>Running Lucene contrib db/bdb task 'get-db-jar' ...</echo>
+    <subant target="get-db-jar">
+      <fileset dir="lucene/contrib/db/bdb" includes="build.xml" />
+    </subant>
+  </target>
+  <target name="finish-idea-setup-windows" if="isWindows">
+    <echo>
+      To install the Lucene/Solr codestyle file, copy
+      dev-tools\idea\Intellij-Lucene-Codestyle.xml to
+      ${env.HOMEDRIVE}${env.HOMEPATH}\.IntelliJIdeaXX\config\codestyles\
+      where "XX" is "90" for IntelliJ 9.0.X, "10" for 10.0.X, etc.
+      After restarting IntelliJ, select "Lucene"
+      from the dropdown list at:
+      Settings | Code Style | Use global settings | Scheme name
+
+      To complete IntelliJ IDEA setup, you must manually configure
+      Project Structure | Project | Project SDK.
+    </echo>
+  </target>
+  <target name="finish-idea-setup-mac" if="isMac">
+    <echo>
+      To install the Lucene/Solr codestyle file, copy
+      dev-tools/idea/Intellij-Lucene-Codestyle.xml to
+      ~/Library/Preferences/IntelliJXX/codestyles/
+      where "XX" is "90" for IntelliJ 9.0.X, "10" for 10.0.X, etc.
+      After restarting IntelliJ, select "Lucene"
+      from the dropdown list at:
+      Settings | Code Style | Use global settings | Scheme name
+
+      To complete IntelliJ IDEA setup, you must manually configure
+      Project Structure | Project | Project SDK.
+    </echo>
+  </target>
+  <target name="finish-idea-setup-unix" if="isUnix">
+    <echo>
+      To install the Lucene/Solr codestyle file, copy
+      dev-tools/idea/Intellij-Lucene-Codestyle.xml to
+      ~/.IntelliJIdeaXX/config/codestyles/
+      where "XX" is "90" for IntelliJ 9.0.X, "10" for 10.0.X, etc.
+      After restarting IntelliJ, select "Lucene"
+      from the dropdown list at:
+      Settings | Code Style | Use global settings | Scheme name
+
+      To complete IntelliJ IDEA setup, you must manually configure
+      Project Structure | Project | Project SDK.
+    </echo>
+  </target> 
+  <target name="clean-idea" 
+          description="Removes all IntelliJ IDEA configuration files">
+    <delete dir=".idea" failonerror="true"/>
+    <delete failonerror="true">
+      <fileset dir="." includes="*.iml,*.ipr,*.iws"/>
+      <fileset dir="solr" includes="**/*.iml"/>
+      <fileset dir="lucene" includes="**/*.iml"/>
+    </delete>
+  </target>
+
   <target name="clean" description="Clean Lucene and Solr">
     <sequential>
       <subant target="clean" inheritall="false" failonerror="true">
